examples/Get_embeddings_from_dataset.ipynb

```diff
@@ -4,6 +4,11 @@
    "cell_type": "markdown",
```

Comment:
Heads up, if we rename this file, we'll have to update all the references to it across other notebooks. 

Check out this search: https://github.com/search?q=repo%3Aopenai%2Fopenai-cookbook+Obtain_dataset&type=code

Reply:
Gotcha - good call. Will make sure to do code search next time. Updated references.